README.md

```diff
@@ -1,2 +1,25 @@
-# org
-Cross-project tasks and organizational items
+# Welcome!
+Welcome to `pnext`. `pnext` is a open open-source project (see: [#org](#org)) focussed on displaying [pointcloud](https://en.wikipedia.org/wiki/Point_cloud) data.
+
+## org
+This project is organized open. All members are part of this github organization. Issue, agendas of the whole project
+are discussed in the [issue](https://github.com/pnext/org/issues) tracker. All documents in this repository are live
+documents - subject to change. You a are encouraged to ask questions and suggest changes on how we do things.
+_(We prefer a [PR](https://help.github.com/articles/about-pull-requests/) over a question though...)_
+
+In the community we currently have two teams:
+ 
+- **admin:** Administrators can create and archive projects.
+    - [Hugo Campos](https://github.com/hccampos)
+    - [Daniel Kastl](https://github.com/dkastl)
+    - [Martin Heidegger](https://github.com/martinheidegger)
+- **developers:** Developers have write access to all repositories.
+
+We generally recommend that any contributor with two PR's join our organization.
+
+## History
+There has been a lot of prior work to the work, displaying pointcloud data. This project original started as fork of [potree](https://github.com/potree/potree) but also draws inspiration from [plasio](https://github.com/hobu/plasio.js),
+[entwine](https://github.com/connormanning/entwine) and [Cesium](https://cesiumjs.org/).
+
+## License
+All projects in this group are to be published under the [MIT](https://en.wikipedia.org/wiki/MIT_License) License.
```

Comment:
@mschuetz just pointed out that Potree is using the [BSD License](https://github.com/potree/potree/blob/develop/LICENSE), which means that any code would need to be either BSD or completely rewritten.

Reply:
Any reason, that it shouldn't be BSD?
Whether it's BSD or MIT, they are both permissive licenses.

Reply:
I added MIT out of simple habit, was a mistake but I just wanted @hccampos's input

Reply:
IANAL but I think they end up being almost the same and I think they are also compatible, so either way is good.